build.gradle

```diff
@@ -49,6 +49,9 @@ apply from: file('shared_versions.gradle')
 def uaaVersion = version
 def geArtifactoryReader = System.getenv('ARTIFACTORY_READER')
 def geArtifactoryReaderPassword  = System.getenv('ARTIFACTORY_READER_PW')
+def proxyHost  = System.getenv('PROXY_HOST')
```

Comment:
If you'd like, a better version of this change is at https://github.com/GESoftware-CF/uaa/commit/6bd8075eb4085c2fbae77722f3798a474f57719f   . I think this also removes the need to change the integrationTest task - please verify.